WalletWasabi/Crypto/ZeroKnowledge/LinearRelation/Equation.cs

```diff
@@ -0,0 +1,61 @@
+using NBitcoin.Secp256k1;
+using System.Linq;
+using WalletWasabi.Crypto.Groups;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge.LinearRelation
+{
+	// Each proof of a linear relation consists of multiple knowledge of
```

Comment:
Not sure why this comment type is used. Typically, people use xmldoc (i.e. `///` comments).

Reply:
only because i'm a C# n00b and i haven't learned that part yet

Reply:
We can consider using xmldoc soon.